frontend/app/tests/e2e/fixtures/account-balances/manual-balances.json

```diff
@@ -17,7 +17,7 @@
   },
   {
     "asset": "USD",
-    "keyword": "United States Dollar",
+    "keyword": "USD",
```

Comment:
There is a reason why the search keyword was different than the asset symbol.

The reason is to test that the autocomplete also works when searching by name too.
If we need to reduce the flakiness, we would have to do that by finding another way, not by changing the test.

Reply:
okay then, i will ping backend to adjust